x-pack/plugins/rule_registry/server/types.ts

```diff
@@ -71,3 +72,11 @@ export interface RacApiRequestHandlerContext {
 export type RacRequestHandlerContext = CustomRequestHandlerContext<{
   rac: RacApiRequestHandlerContext;
 }>;
+
+export type BrowserField = FieldSpec & {
```

Comment:
I am wondering if we should move these type under the common folder so we can re-use it

Reply:
Yes, I'll move it in the frontend PR